data/topic-schema/src/main/resources/net/corda/schema/Cluster.yaml

```diff
@@ -0,0 +1,18 @@
+topics:
+  ClusterWorkerInfo:
```

Comment:
Shouldn't this be a compacted topic?

Reply:
I was expecting it to be PubSub.  As I understand it, P2P was going to need it to know the minimum version on the system and Compacted would hide if some worker was somehow unavailable (turned off or decommissioned),

Though, maybe that's not a terribly likely scenario for the workers?

Reply:
I assumed the worker ID would maybe be part of the key. 
If it's pub/sub, how would a new consumer get the full list?
What's the schema for the message on the topic?